uber/automated_emails.py

```diff
@@ -290,14 +290,6 @@ def __init__(self, subject, template, filter, ident, **kwargs):
         needs_approval=True,
         ident='dealer_reg_payment_reminder_last_chance')
 
-    MarketplaceEmailFixture(
-        '{EVENT_NAME} Dealer waitlist has been exhausted',
-        'dealers/waitlist_closing.txt',
-        lambda g: g.status == c.WAITLISTED,
-        # query=Group.status == c.WAITLISTED,
-        when=after(c.DEALER_WAITLIST_CLOSED),
-        ident='uber_marketplace_waitlist_exhausted')
```

Comment:
Is this something we want to delete?

Reply:
Yea, this email hasn't been sent in at least two years. It trips the "email ready to send" reminder forever, which was annoying to the people getting those emails.

Reply:
Good to know!